server/views/pages/accessibilityStatement.njk

```diff
@@ -0,0 +1,13 @@
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Accessibility statement" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">Accessibility statement</h1>
```

Comment:
I'll put the actual content of this (and the other static pages) in next PR - this one is just to get the mechanism of them working in.